src/Utils/obiDialogParser.ts

```diff
@@ -182,14 +189,14 @@ export class ObiDialogParser {
     /**
      * Collects dialog nodes from dialog-redirecting elements in the dialog `steps` section.
      */
-    private async collectDialogStepChildren(node: ObiDialogNode, steps: (string | OBITypes.OBIDialog)[]) {
+    private async collectDialogStepChildren(node: ObiDialogNode, steps: (string | OBITypes.OBIDialog)[],
+        conditionalEntities: { [key: string]: Set<string> }) {
```

Comment:
I've seen this is multiple reviews and didn't get answers on the other ones. Maybe better to ask why are you doing this?
Sometime it's an argument type other times it's a return type. Doesn't seem to be anything special about this one argument. I can't make sense of it other than something like an 80 character width limit rule.

Reply:
I have my editor split into 2 panes, so I'm generally just breaking up these long lines if they don't fit in the pane on ~half my monitor.

Updating to args on single lines per your earlier request.

Reply:
It sounds like a problem with using word wrapping instead of actually putting a newline. This way you can satisfy personal preferences / conditions of small screen without influencing code expectations for everyone else.  Good to have separate lines for these especially long arguments names and types as opposed to something small like `index: number`